Formula/petsc.rb

```diff
@@ -0,0 +1,43 @@
+class Petsc < Formula
+  desc "Scalable solution of models that use partial differential equations"
+  homepage "https://www.mcs.anl.gov/petsc/index.html"
+  url "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.9.0.tar.gz"
+  sha256 "a233e0d7f69c98504a1c3548162c6024f7797dde5556b83b0f98ce7326251ca1"
+
+  depends_on "gcc"
+  depends_on "hdf5"
+  depends_on "hwloc"
+  depends_on "metis"
+  depends_on "netcdf"
+  depends_on "open-mpi"
+  depends_on "scalapack"
+  depends_on "suite-sparse"
+
+  def install
+    system "./configure", "CC=mpicc",
+                          "CXX=mpicxx",
+                          "F77=mpif77",
+                          "FC=mpif90",
+                          "--prefix=#{prefix}",
+                          "--with-debugging=0",
+                          "--with-scalar-type=real",
+                          "--with-shared-libraries=1",
+                          "--with-ssl=0",
```

Comment:
why

Reply:
It is the default. SSL is only used for an experimental feature that can deliver SMS notification of job completion. The two lines above are defaults and can be deleted.